cedars/app/db.py

```diff
@@ -496,12 +495,13 @@ def get_documents_to_annotate(patient_id=None):
     return documents_to_annotate
 
 
-def get_all_annotations_for_patient(patient_id):
+def get_all_annotations_for_patient(patient_id, unique_sentences = True):
```

Comment:
In this function:
1. We want to not show "same" sentences across notes. The clinician typically copy pastes notes from one note to the other - so we want to show the sentence the first time it occurs and not again. 
2. pleaase make sure that the order in which the sentences are shown is correct i.e. the reviewer should see the note sorted by time (ascending)